src/frontend/src/features/home/routes/Home.tsx

```diff
@@ -32,10 +42,10 @@ export const Home = () => {
               variant="primary"
               onPress={
                 isLoggedIn
-                  ? () =>
-                      navigateTo('room', generateRoomId(), {
-                        state: { create: true },
-                      })
+                  ? async () => {
+                    const slug = generateRoomId()
+                    await createRoom({slug})
+                  }
```

Comment:
can we write:

```js
isLoggedIn
  ? () => createRoom({slug: generateRoomId()})
  : undefined
```

?

Reply:
I'd even argue, since the user doesn't choose its room name, that the homepage shouldn't have to `generateRoomId`, `createRoom` could internally generate the id.

Reply:
Yes, but users will soon have the option to either generate a random ID or type a room name. I believe the homepage should handle this responsibility, allowing the `createRoom` function to be extended accordingly.